### PR TITLE
hw_1_1

### PR DESCRIPTION
**Pull request by IrinaDyakonova**

### CODE REVIEW

**Comment by dmytro-hryhoruk** [comment]
Do we need so many '?' conditions?

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 1]
```diff
@@ -15,6 +15,9 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+
+        return (result1 == bet1 && result2 == bet2)
+               ? 2 : ((bet2 > bet1 && result2 > result1) || (bet1 > bet2 && result1 > result2)
+                || (result1 == result2 && bet1 == bet2)) ? 1 : 0;
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java]:
```suggestion
        return (result1 == bet1 && result2 == bet2)
               ? 2 : ((bet2 > bet1 && result2 > result1) 
                || (bet1 > bet2 && result1 > result2)
                || (result1 == result2 && bet1 == bet2))  
               ? 1 : 0;
```

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 2]
```diff
@@ -15,6 +15,11 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/FootballMatch.java:18]:
```suggestion
```

**Review by dmytro-hryhoruk — APPROVED** [review]

File: src/main/java/core/basesyntax/FootballMatch.java (hunk 2 above)
Comment (IrinaDyakonova) [reply src/main/java/core/basesyntax/FootballMatch.java:18]: public int matchResult(int result1, int result2, int bet1, int bet2) {

        return (result1 == bet1 && result2 == bet2)
                ? 2 : ((bet2 > bet1 && result2 > result1)
                || (bet1 > bet2 && result1 > result2)
                || (result1 == result2 && bet1 == bet2))
                ? 1 : 0;
    }

нужно убрать пустую строку?

**Pull request closed without merging**